_getting_started/installation/index.md

```diff
@@ -4,35 +4,120 @@ layout: default
 weight: -9
 ---
 
-The following installation instructions work for Linux, MacOS and
-Windows.
+<style>
+  .marker { margin-bottom: 40px;}
+</style>
 
-There is an <a
-href="https://davidgranstrom.github.io/tidal-bootstrap/">automated
-script for MacOS X users</a>, along with a <a
-href="https://www.youtube.com/watch?v=dJTfGv2sT-w&t=2s">helpful
-walkthrough video</a>, as an alternative to the below.
+You can install TidalCycles on MacOS, Windows, and Linux. Locate the instructions
+below for your specific operating system.
 
-## If you have install issues
+* [MacOS](#macos)
+* [Windows](#windows)
+* [Linux](#linux)
 
-There appears to be an issue with Cabal that is affecting installation.  If you are unable to install TidalCycles with the directions below, please try these alternate installation instructions.
+<a id="macos">&nbsp;</a>
+<hr class="marker">
 
-From a terminal window, run this:
+## Mac OS 
+
+### 1. Prerequisites
+
+You first need to install the following three pieces of software,
+Haskell, Atom, and SuperCollider. You can download them from their
+websites - click on each of the below:
+
+* [Haskell](https://www.haskell.org/platform/)
+* [Atom](https://atom.io/)
+* [SuperCollider](http://supercollider.github.io/download) (version 3.7 or later)
+* You may also need to install [Git](https://git-scm.com/)
+
+You will need the SuperCollider sc3-plugins for using many of the synths included in 
+SuperDirt. Most of the examples in the documentation will still work, so you could
+skip this step and return to it later. You can install the latest version from
+[github](https://github.com/supercollider/sc3-plugins) according to
+the instructions there.
+
+### 2. Install the TidalCycles pattern engine
+
+>**NOTE** As of September 26 2017, There appears to be an issue with Cabal that is 
+>affecting installation on MacOS.  
+>If you are unable to install TidalCycles with the directions below, please try 
+>these alternate installation instructions. From a terminal window, run this:
+
+>~~~~bash
+>stack setup
+>stack install tidal
+>~~~~
+
+>Once that's done, you'll also need to change a configuration setting
+>in Atom. Find the package settings for TidalCycles (*preferences >
+>tidalcycles > settings*). Change the ```'ghci path'``` option to this:
+
+>```stack ghc -- --interactive```
```

Comment:
We could just build this into the Atom plugin rather than add an extra step...?

See https://github.com/tidalcycles/atom-tidalcycles/issues/23

Reply:
I think we're only telling MacOS users to use Stack at this point. I'm not sure we want to change the Atom plugin for Windows and Linux and use Stack on those OS's yet.

Reply:
ah, I thought I was doing the right thing here by removing `-XOverloadedStrings` only, but the `-- --interactive` can be removed too. The Atom plugin has already been changed. Thanks for catching this @jarmitage .